OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -103,7 +103,7 @@ public override Activity Tick(Actor self)
 				foreach (var n in notify)
 					n.MovingToResources(self, closestHarvestablePosition.Value, this);
 
-				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 1), new HarvestResource(self), this);
+				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 2), new HarvestResource(self), this);
```

Comment:
https://www.youtube.com/watch?v=Zl7j4hSTW6g : As you can see in my manual induction of harvester getting stuck, I don't think it is 100% closestHarvestablePosition's bug. At the moment of path finding, the ore wasn't blocked.